src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/PreferStreamReadAsyncMemoryOverloadsTests.cs

```diff
@@ -122,9 +122,9 @@ public Task CS_Analyzer_NoDiagnostic_Stream_NoAwait_ReturnMethod()
 using System.Threading.Tasks;
 class C
 {
-    public Task M(Stream s, byte[] buffer)
+    public void M(Stream s, byte[] buffer)
     {
-        return s.ReadAsync(buffer, 0, 1);
+        s.ReadAsync(buffer, 0, 1);
```

Comment:
The test is named "...ReturnMethod"; presumably you want to change that now that the case is invoking ReadAsync without returning it (or awaiting it)?

Reply:
I was _just_ fixing that before you commented this. You read my mind 😄

Reply:
Fixed